src/planner.cc

```diff
@@ -52,6 +52,34 @@ struct AxisTarget {
 };
 }  // end anonymous namespace
 
+// Given the desired target speed(or acceleration) of the defining axis and the steps to be
+// performed on all axes, determine if we need to scale down as to not exceed
+// the individual maximum speed or acceleration constraints on any axis. Return the new
+// defining axis limit based on the euclidean motion requested.
+//
+// Example with speed: given i the axis with the highest (relative) out of
+// bounds speed, what's the speed of the defining_axis so that every speed
+// respects i's bounds? We just need to project each speed to the defining
+// axis speed, and pick the smallest.
+//
+// We need to work in mm, since steps is not a uniform unit of space across
+// the different axes.
+static float clamp_defining_axis_limit(const int *axes_steps,
+                                    const FloatAxisConfig &axes_limits_mm,
```

Comment:
This indentation looks off. I just checked, the clang-format was apparently an older version.
Can you re-base to head run `.github/bin/run-clang-format.sh` ?

Reply:
(solved)